tests/unit/test_models/test_model_info.py

```diff
@@ -1,12 +1,10 @@
 #
 # Tests getting model info
 #
-from tests import TestCase
 import pybamm
-import unittest
 
 
-class TestModelInfo(TestCase):
+class TestModelInfo:
```

Comment:
I hope the discussion on Slack answers your question about this test!

Reply:
Yes thanks!